cvat-ui/src/reducers/useragreements-reducer.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { boundariesActions, BoundariesActionTypes } from 'actions/boundaries-actions';
+import { UserAgreementsActions, UserAgreementsActionTypes } from 'actions/useragreements-actions';
+import { AuthActions, AuthActionTypes } from 'actions/auth-actions';
+import { UserAgreementsState } from './interfaces';
+
+const defaultState: UserAgreementsState = {
+    list: [],
+    fetching: false,
+    initialized: false,
+};
+
+export default function (
+    state: UserAgreementsState = defaultState,
+    action: UserAgreementsActions | AuthActions | boundariesActions,
+): UserAgreementsState {
+    switch (action.type) {
+        case UserAgreementsActionTypes.GET_USER_AGREEMENTS: {
+            return {
+                ...state,
+                fetching: true,
+                initialized: false,
+            };
+        }
+        case UserAgreementsActionTypes.GET_USER_AGREEMENTS_SUCCESS:
+            return {
+                ...state,
+                fetching: false,
+                initialized: true,
+                list: action.payload,
+            };
+        case UserAgreementsActionTypes.GET_USER_AGREEMENTS_FAILED:
```

Comment:
I would suggest to add this action also to notification-reducer (like for formats, etc.).
Otherwise if request is wrong, we do not see any notification about it and do not see any agreements on the register page